### PR TITLE
CLN: Remove io.data and io.wb

### PR DESCRIPTION
**Pull request by gfyoung**
Deprecated in 0.17.0.

xref #13735.

### CODE REVIEW

**Comment by jreback** [comment]
any remaining docs ?

**Comment by gfyoung** [comment]
Couldn't find anything that needed changing via GitHub search.

**Comment by jreback** [comment]
http://pandas-docs.github.io/pandas-docs-travis/remote_data.html

**Comment by gfyoung** [comment]
What about it?  There's nothing out-of-date with this comment.

**Comment by jreback** [comment]
let’s remove that section

**Comment by gfyoung** [comment]
So just delete `remote_data.rst` from the codebase?

File: doc/source/whatsnew/v0.22.0.txt [hunk 1]
```diff
@@ -136,6 +136,7 @@ Removal of prior version deprecations/changes
 - ``pd.tseries.util.pivot_annual`` has been  removed (deprecated since v0.19). Use ``pivot_table`` instead (:issue:`18370`)
 - ``pd.tseries.util.isleapyear`` has been removed (deprecated since v0.19). Use ``.is_leap_year`` property in Datetime-likes instead (:issue:`18370`)
 - ``pd.ordered_merge`` has been removed (deprecated since v0.19). Use ``pd.merge_ordered`` instead (:issue:`18459`)
+- The ``pandas.io.wb`` and ``pandas.io.data`` modules have been removed (:issue:`13735`)
```
[the remark this answers is not in the document]
Comment (gfyoung) [reply doc/source/whatsnew/v0.22.0.txt]: What exactly would you say?  It's a little tricky to describe the difference between now and then with regards to this change.  To be more precise about what changed is perhaps more relevant for a developer than the everyday user to be honest.

[the remark this answers is not in the document]
Comment (jorisvandenbossche) [reply doc/source/whatsnew/v0.22.0.txt]: Well, the exact thing that changes is actually only the *message* of the import error, so maybe not worth to list at all?

[the remark this answers is not in the document]
Comment (gfyoung) [reply doc/source/whatsnew/v0.22.0.txt]: Perhaps not, actually.  @jreback , what do you think?

[the remark this answers is not in the document]
Comment (jreback) [reply doc/source/whatsnew/v0.22.0.txt]: these are really stub modules, which you could say.

**Comment by jreback** [comment]
remove remote_data.rst entirely as well.

File: doc/source/whatsnew/v0.22.0.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (gfyoung) [reply doc/source/whatsnew/v0.22.0.txt]: Fair enough, done.

**Comment by jreback** [comment]
thanks!

**Pull request merged**